api/envoy/config/bootstrap/v2/bootstrap.proto

```diff
@@ -155,6 +155,11 @@ message Bootstrap {
   // <server_statistics>` if specified. Envoy will not process this value, it will be sent as is to
   // :ref:`stats sinks <envoy_api_msg_config.metrics.v2.StatsSink>`.
   google.protobuf.UInt64Value stats_server_version_override = 19;
+
+  // Always use TCP queries instead of UDP queries for DNS lookups.
+  // This may be overridden on a per-cluster basis in cds_config,
+  // when dns_resolvers and use_tcp_for_dns_lookups are specified.
```

Comment:
Nit: can you `:ref:` link this to the field documentation?

Reply:
done

Reply:
Looks good, ready to merge once the other nits are cleaned up.